src/core/surface/init.c

```diff
@@ -64,7 +64,7 @@ void grpc_init(void) {
     grpc_security_pre_init();
     grpc_iomgr_init();
     grpc_tracer_init("GRPC_TRACE");
-    census_init();
+    census_initialize(CENSUS_NONE);
```

Comment:
Originally this was pragmatic while I was moving old->new files/API. Now, I'd rather keep the full name; since census is supposed to be (mostly) independent, think it should be OK. Will change if people feel strongly.